components/EntireScheme.svelte

```diff
@@ -69,7 +71,17 @@
     // TODO Should we prompt before deleting the current scheme?
     reader.onload = (e) => {
       try {
-        gjScheme.set(JSON.parse(e.target.result));
+
+        let result = JSON.parse(e.target.result);
+
+        // test if origin field is set
```

Comment:
Style nit-picks:

1) In the final code we check in, I would vote against these comments, since they just repeat what the code says. A helpful thing to comment would be why checking for this field is sufficient to decide this comes from ATIP
2) If you run `npm run fmt`, it'll add some standardized spacing
3) Don't we want to check `result.origin` starts with `atip-`? It's unlikely someone has a geojson file with this [foreign member](https://www.rfc-editor.org/rfc/rfc7946#section-6) set, but still seems useful

Reply:
Oops, I just realized `npm run fmt` is assuming a globally installed copy of `prettier` exists. I'll fix that...

Reply:
If you sync past 50bc1d0126b1dd28643fc1bd7c424ad34be56f8a, `npm run fmt` now does the right thing :)

Reply:
Merged my branch up past 50bc1d0 and run `npm run fmt`

I've also added an or condition to check for `authority` to allow for backwards compatibility.

On point 3, yes we probably should do that. I couldn't implement it nicely just now (can't check the string if `origin` doesn't exist to be backwards compatible) but will have a think on it.